e2e_tests/tests/experiment/test_allocation_csv.py

```diff
@@ -48,15 +48,20 @@ def test_notebook_capture() -> None:
 
         for line in notebook.stdout:
             if re.search("Jupyter Notebook .*is running at", line) is not None:
-                return
+                break
+
     assert task_id is not None
 
     end_time = datetime.datetime.now(datetime.timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
     sess = api_utils.user_session()
     r = sess.get(f"{API_URL}timestamp_after={start_time}&timestamp_before={end_time}")
     assert r.status_code == requests.codes.ok, r.text
 
-    assert re.search(f"{task_id},NOTEBOOK", r.text) is not None
+    assert re.search(f"{task_id}.*,NOTEBOOK", r.text) is not None
+
+    workspace = clu.utils.get_task_info(sess, "notebook", task_id).get("workspaceName", None)
```

Comment:
For this import: `from tests import cluster as clu`

Can you replace it with `from tests.cluster import utils as cluster_utils` or something similar?

We try to use the lowest level module name when we can, e.g. `module.function()` is preferable to `module.module.function()`